market/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from .models import TRANSACTION_MODES
+
+
+class StockTransactionForm(forms.Form):
+    mode = forms.ChoiceField(choices=TRANSACTION_MODES)
+    quantity = forms.CharField(required=True,
+                               widget=forms.TextInput(attrs={
+                                   'class': 'form-control',
+                                   'placeholder': 'Enter integers only',
+                                   'title': 'Enter integers only',
+                                   'pattern': '[0-9]+'
+                               }))
```

Comment:
The quantity variable line should be written in the following way according to PEP 8 guidelines.

```[python]
quantity = forms.CharField(
    required=True,
    widget=forms.TextInput(attrs={
        'class': 'form-control',
        'placeholder': 'Enter integers only',
        'title': 'Enter integers only',
        'pattern': '[0-9]+'
    })
)
```